server/src/test/java/io/druid/client/DirectDruidClientTest.java

```diff
@@ -314,7 +313,7 @@ public void testQueryInterruptionExceptionLogMessage() throws JsonProcessingExce
 
     TimeBoundaryQuery query = Druids.newTimeBoundaryQueryBuilder().dataSource("test").build();
     HashMap<String, List> context = Maps.newHashMap();
-    interruptionFuture.set(new ByteArrayInputStream("{\"error\":\"testing\"}".getBytes()));
+    interruptionFuture.set(new ByteArrayInputStream("{\"error\":\"testing1\",\"errorMessage\":\"testing2\"}".getBytes()));
```

Comment:
In the current change `testing1` gets swallowed. I'm not sure it was previously possible to inject something like `testing` into the error field, but it seems odd that such a thing would be completely swallowed and converted to `Unknown exception`.

In general I'm good with limiting the `error` field to have only certain values. Hopefully noone out there has custom error codes that this will break.